install.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+PREFIX=/home/taste/tool-inst
+SOURCES=$(dirname $0)
+
+mkdir -p "${PREFIX}/include/TASTE-LEON3-Drivers"
+mkdir -p "${PREFIX}/include/TASTE-LEON3-Runtime/src"
+
+rm -rf "${PREFIX}/include/TASTE-LEON3-Drivers/src/*"
+cp -r "${SOURCES}/src/" "${PREFIX}/include/TASTE-LEON3-Drivers/src/"
+cp -r "${SOURCES}/configurations" "${PREFIX}/include/TASTE-LEON3-Drivers/configurations"
+
+cp -r "${SOURCES}/TASTE-LEON3-Runtime/Leon3-BSP" "${PREFIX}/include/TASTE-LEON3-Runtime/src/"
```

Comment:
We will have to check whether this does not collide with Runtime installation.

Reply:
It is, but it's only helper script, not used except for possible development. Should I delete it?

Reply:
If it is unused, then delete it. If it is used (in what manner then?), make sure that it is consistent.

Reply:
Removed